CalibCalorimetry/CastorCalib/src/CastorLedAnalysis.cc

```diff
@@ -20,6 +20,7 @@ CastorLedAnalysis::CastorLedAnalysis(const edm::ParameterSet& ps)
   evt=0;
   sample=0;
   m_file=nullptr;
+  char output[100]{0};
```

Comment:
Correct, trying to keep it minimal here, somewhat. If you want to do a bigger refactoring it would be better if responsible people/authors do it.

I haven't even looked where `char output[]` ends up being used at the end.